Library/Homebrew/commands.rb

```diff
@@ -30,6 +30,13 @@ module Commands
     "lc"          => "livecheck",
     "tc"          => "typecheck",
   }.freeze
+  HOMEBREW_UNSUPPORTED_COMMAND_MESSAGES = {
+    "cask" => "`brew cask <command>` is no longer supported. Use `brew <command> --cask` instead.",
+  }.freeze
+
+  def unsupported_cmd?(cmd)
```

Comment:
Methods with a trailing `?` should always return a boolean rather than a string.